reana_client/cli/workflow.py

```diff
@@ -298,6 +299,9 @@ def workflow_create(ctx, file, name, skip_validation, access_token):  # noqa: D3
         reana_specification = load_reana_spec(
             click.format_filename(file), skip_validation
         )
+        click.echo(reana_specification)
+        if not check_htcondor_max_runtime(reana_specification):
```

Comment:
Since this implementation is Serial specific, what do you think about moving the check to the [`load_serial` function](https://github.com/reanahub/reana-commons/blob/34ddc1ef866b0f54eba1789153f8001d107d234b/reana_commons/serial.py#L82)?

REANA-Client will eventually execute it through [`load_reana_spec` function](https://github.com/reanahub/reana-client/blob/2436dde2df70ca835afcc49ad81f1f91ce79e030/reana_client/utils.py#L112) which will perform a validation of the [`reana.yaml`](https://github.com/reanahub/reana-client/blob/2436dde2df70ca835afcc49ad81f1f91ce79e030/reana_client/utils.py#L149) in general and [the workflow specifications in particular depending on its type](https://github.com/reanahub/reana-client/blob/2436dde2df70ca835afcc49ad81f1f91ce79e030/reana_client/utils.py#L152-L156).

Reply:
Also, this means that we would be missing this check for Yadage and CWL, for that we can create a couple of issues to not overload this PR group?

Reply:
Ooh of course this will only work for Serial! These values are not set in the reana specifications for CWL and Yadage.

Reply:
> Also, this means that we would be missing this check for Yadage and CWL, for that we can create a couple of issues to not overload this PR group?

Yes I agree.

Reply:
Closed this PR as the function has been moved to r-commons with this commit reanahub/reana-commons@985f71f9a57fa79fe9ec4a538d79630bf7b8eae9.